<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove backgrounds</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="best">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Best</string>
    </property>
    <widget class="QRadioButton" name="bigben">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>108</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Big Ben</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>108</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Cat</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="duck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>108</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Duck</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="tomato">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>108</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Tomato</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="trafficlight">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>108</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Traffic Light</string>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="slider_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>21</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="value_lcd">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>30</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>319</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
